check_phpfpm_status.pl

```diff
@@ -107,6 +108,8 @@ sub help {
    ServerName, (host header of HTTP request) use it if you specified an IP in -H to match the good Virtualhost in your target
 -f, --fastcgi
    Connect directly to php-fpm via network or local socket, using fastcgi protocol instead of HTTP.
+-u, --unixsocket
```

Comment:
The `-u` option is already used as a shortcut for `--url`. My perl is a bit rusty, but it looks like there is no short form for the `--unixsocket` option at all?

Reply:
thx, you are right. The problem is that the most useful short forms are already in use (-u -U -s -S). I'm considering to give --unixsocket no short form (like --verifyssl). Someone else having an idea? :)

Reply:
Personally, I am happy with the long form - it's an icinga plugin, so not something we need to type regularly :)